runtime/vflow/src/configs.rs

```diff
@@ -20,3 +20,25 @@ mod governance;
 pub mod monetary;
 pub mod system;
 pub mod xcm;
+
+#[cfg_attr(not(feature = "volta"), path = "configs/mainnet.rs")]
```

Comment:
Consider to use `cfg_if` crate here... It's my taste, but I dont like _if feature / if not feature_ patters

Reply:
Ok done